src/app/buzz.js

```diff
@@ -0,0 +1,48 @@
+import { sessionReducer } from "../session/session";
+import { applyMiddleware, createStore } from "redux";
+import { loadState, saveState } from "../persistence/storePersistence";
+import { throttle } from "lodash";
+import { RESET_STATE, resetState } from "./actions";
+import testUtils from "../testUtils/utils";
+import thunk from "redux-thunk";
+
+const buzzStore = createStore(
+  buzzReducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+  applyMiddleware(thunk)
+);
+configurePersistence(buzzStore);
+
+function configurePersistence(store) {
+  return loadInitialState(store).then(() => {
+    configureSubscriberToPersistStateChanges(store);
```

Comment:
Loads the initial state asynchronously and configures the store to save the state change using a 1 second throttle so we don't impact the performance after every state change.